CHANGES.rst

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-17)
+--------------------------
+
+- Makes workflow engine independent of Celery so that independent workflow
+  instances are created on demand for each user.
+- Centralises the initialisation of ``WorkflowStatusPublisher``.
+- Introduces CVMFS mounts in job specifications.
+- Makes docker image slimmer by using ``python:2.7-slim``.
+- Centralises log level and log format configuration.
+- Sets the same ``click`` and ``pytest`` requirements for all REANA components.
```

Comment:
See the comment about last phrase in the Serial PR.  BTW I think this phrase is missing in the CWL PR, perhaps because it is outdated... that one needs rebasing